### ISSUE TITLE
Request to revert https://github.com/checkstyle/checkstyle/issues/17551

### ISSUE THREAD

**Issue by lrozenblyum** [checkstyle/checkstyle#18616]
https://github.com/checkstyle/checkstyle/issues/17551 looks to be an incorrect false negative report.

According to https://docs.oracle.com/javase/specs/jls/se21/html/jls-3.html#jls-3.10.7 `\s` **escape sequence** is representing just `\u0020`.

Whitespace characters that cause now violations due to #17551 have NO corresponding escape sequences.

Consider reverting #17551 because now it causes unsolvable false positives.

**Comment by giruuuuj** [comment]
# Issue #18616: Revert PR #17551 to fix false positives

## 🚨 **False Positive Fix**

### **Problem Summary**
PR #17551 introduced incorrect false positives by treating various Unicode whitespace characters as if they should use `\s` escape sequence.

### **Root Cause Analysis**
- **JLS Specification**: According to [JLS §3.10.7](https://docs.oracle.com/javase/specs/jls/se21/html/jls-3.html#jls-3.10.7), `\s` escape sequence represents **only** `\u0020` (space)
- **False Assumption**: PR #17551 incorrectly assumed `\s` represents all Unicode whitespace characters
- **Reality**: Unicode characters like `\u000b`, `\u000c`, `\u1680`, etc. have **NO corresponding escape sequences**

### **Impact**
- **Before PR #17551**: Correct behavior - no false positives
- **After PR #17551**: Unsolvable false positives for valid Unicode usage
- **After This Revert**: Restores correct behavior

### **Changes Made**

#### 1. **Regex Pattern Restoration**
```xml
<!-- Reverted to original pattern that only flags actual escape sequence violations -->
<property name="format" 
    value="\\u00(09|0(a|A)|0(c|C)|0(d|D)|22|27|5(C|c))|\\(0(10|11|12|14|15|42|47)|134)"/>

**Comment by romani** [comment]
we need to fix forward.
@vivek-0509 , can you help us to fix this issue ?

**Comment by romani** [comment]
@lrozenblyum , please report issue by issue template that we demand to follow, it help people to easily understand problem and provide a fix.

**Comment by vivek-0509** [comment]
@romani Sure will fix this, please approve the issue

**Comment by romani** [comment]
will be fixed at https://github.com/checkstyle/checkstyle/pull/18757
